cluster/manifests/kube-janitor/rbac.yaml

```diff
@@ -0,0 +1,41 @@
+{{ if ne .Environment "production" }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: kube-janitor
+  namespace: kube-system
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: kube-janitor
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - events
+  verbs:
+  - create
```

Comment:
Read operations as well, plus update/patch, so coalescing could work.

Reply:
Added patch and update for events. It can already read and delete ALL resources. Maybe we want to define a whitelist but this is how it's supposed to work afaik. I would not do the white list in this PR if we go that way.